addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -4,6 +4,7 @@ odoo.define('wysiwyg.widgets.media', function (require) {
 var core = require('web.core');
 var Dialog = require('web.Dialog');
 var fonts = require('wysiwyg.fonts');
+var session = require('web.session');
```

Comment:
Merging fja's PR in saas-12.2 which fixes this: https://github.com/odoo/odoo/pull/33247

Reply:
Ok I'll wait forward port.